documentation/staging/content/faq/newbie.md

```diff
@@ -0,0 +1,139 @@
+
+The operator does not specify how a WebLogic domain home configuration is created. You can use WLST, REST, or a very convenient new tool called [WebLogic Server Deploy Tooling](https://github.com/oracle/weblogic-deploy-tooling) (WDT). WDT allows you to compactly specify WebLogic configuration and deployments (including JMS, data sources, applications, authenticators, and such) using a YAML file and a ZIP file (which include the binaries). The operator [samples]({{< relref "/samples/simple/_index.md" >}}) show how to create domains using WLST and using WDT.
+
+
+**Q:** Is the Administration Server required? Node Manager?
+
+**A:** Certification of both WebLogic on Docker and WebLogic in Kubernetes consists of a WebLogic domain with the Administration Server. The operator configures and runs Node Managers for you within a domain's pods - you don't need to configure them yourself - so their presence is largely transparent.
+
+***
+
+#### Communications
+
+**Q:** How is location transparency achieved and the communication between WLS instances handled? T3s?
+
+**A:** Inside the Kubernetes cluster, we use the cluster IP (acts as a DNS name) and ingress controller, which allows for the pods with WebLogic Servers to move to different nodes in the Kubernetes cluster and continue communicating with other servers.  
+
+For T3 communication outside the Kubernetes cluster, we use NodePort and configure a WebLogic channel for T3 RMI.  See the blog, [T3 RMI Communication for WebLogic Server Running on Kubernetes](https://blogs.oracle.com/weblogicserver/t3-rmi-communication-for-weblogic-server-running-on-kubernetes), which explains RMI T3 communication in detail. All QoS are supported for T3 and HTTP communication (SSL, administration port).
+
+**Q:** Are clusters supported on Kubernetes using both multicast and unicast?
+
+**A:** Only unicast is supported. Most Kubernetes network fabrics do not support multicast communication. Weave claims to support multicast but it is a commercial network fabric.  We have certified only on Flannel, which supports unicast only.
+
+
+**Q:** Docker resources: We use an SSL listener, SSL port, and an administration port on each instance. How do we map these resources?
+
+**A:** Yes, you can use an SSL listener, SSL port, and an administration port in both Docker and Kubernetes.  See to the blog, [T3 RMI Communication for WebLogic Server Running on Kubernetes](https://blogs.oracle.com/weblogicserver/t3-rmi-communication-for-weblogic-server-running-on-kubernetes), which describes how to map these ports in Kubernetes.
+
```

Comment:
@rjeberhard This question seems to be Docker specific. I'm inclined to leave it as such unless you tell me otherwise.

Reply:
Let's remove this question and answer too.... By the way, where did these come from? Many are really useful, but then others sound "ancient."

Reply:
Some are. They came from the wiki FAQ, https://github.com/oracle/weblogic-kubernetes-operator/wiki/FAQ